test/integration/developers_cant_add_others_to_tribe_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.